pkg/commands/import_params.go

```diff
@@ -0,0 +1,180 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"maps"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/manifoldco/promptui"
+	"github.com/massdriver-cloud/mass/pkg/bundle"
+	"github.com/massdriver-cloud/mass/pkg/provisioners"
+	yaml3 "gopkg.in/yaml.v3"
+)
+
+func ImportParams(buildPath string, skipVerify bool) error {
```

Comment:
This function is currently too cluttered. Some of this functionality will be broken out into functions when we add capability like `mass bundle add-connection` or `mass bundle add-artifact`. Working with yaml nodes is not easy, but it does maintain structure and order in yaml files, allowing us to read and render with minimal extraneous changes.